### ISSUE TITLE
Confusables logic does not evaluate combinations of letters

### ISSUE THREAD

**Issue by plasmacorral** [MetaMask/metamask-mobile#5415]
### **Description**
I saw metarnask.eth was available on ENS and registered it.  I notice that our confusables logic pops on the first "m".  However, the confusable part of this address is the combination of "rn" masquerading as a second "m".

The unicode [confusables](https://util.unicode.org/UnicodeJsps/confusables.jsp?a=metarnask&r=None), highlights rn as the most prominent confusable character for the first "m" and on mobile our warning (first screenshot) even specifically mentions the confusability of "m" for "rn".  

However, we do not highlight the actual 'rn" being used here. This is an opportunity on both extension and mobile.

### **Technical Details**

-   We should look to improve the confusables logic to alert on combinations of letters.

### **Acceptance Criteria**

-   When the user attempts to send to an ENS name that includes a combination of confusable characters
-   Then a confusables warning should be presented
-   And we should highlight the character combination that triggers the warning

Scenario: xxxx
- GIVEN a user is sending to an ENS address like metarnask.eth
- WHEN a user enters a series of characters that could be confusable
- AND a user taps 'Next" on the "Send to" page
- AND the user enters an amount and taps "Next"
- THEN the confirmation page highlights both single confusable character "m" as well as combinations like "rn"

### **References**

-  Mobile screenshots v5.12 build 1024 with iOS 16, iPhone 13 mini:
<div style="display: inline-block"> <img width="190" src="https://user-images.githubusercontent.com/32695229/208763780-b606612f-8725-4f3a-939a-c9cceb2b4998.png"><img width="188" src="https://user-images.githubusercontent.com/32695229/208764497-355b3b80-9a83-49fa-9545-41a8fdab6189.png"></div>

-Extension v 10.22.2 on mac ventura 13.0.1 with firefox 108.0.1:
<div style="display: inline-block"> <img width="190" src="https://user-images.githubusercontent.com/32695229/208764826-33230208-1d9e-451a-a093-e648efe5d569.png"><img width="186" src="https://user-images.githubusercontent.com/32695229/208764874-0c7bd16e-9bc5-43f5-bdaf-558c8d73e56a.png"></div>

**Comment by 7amdok4312007-cmyk** [comment]
> ``9758af24300c325a30b2368d1e5741e4adbc6de3
